RevenueCat/Scripts/PurchasesWrapperAndroid.cs

```diff
@@ -181,5 +182,70 @@ public void CheckTrialOrIntroductoryPriceEligibility(string[] productIdentifiers
         }
     }
 
+    public void InvalidatePurchaserInfoCache()
+    {
+        using (var purchases = new AndroidJavaClass("com.revenuecat.purchasesunity.PurchasesWrapper"))
+        {
+            purchases.CallStatic("invalidatePurchaserInfoCache");
+        }
+    }
+
+    public void SetAttributes(Dictionary<string, string> attributes)
+    {
+        var request = new DictionaryRequest(attributes);
+
+        using (var purchases = new AndroidJavaClass("com.revenuecat.purchasesunity.PurchasesWrapper"))
+        {
+            purchases.CallStatic("setAttributes", JsonUtility.ToJson(request));
+        }
+    }
+
+    public void SetEmail(string email)
+    {
+        using (var purchases = new AndroidJavaClass("com.revenuecat.purchasesunity.PurchasesWrapper"))
+        {
+            purchases.CallStatic("setEmail", email);
+        }
+    }
+
+    public void SetPhoneNumber(string phoneNumber)
+    {
+        using (var purchases = new AndroidJavaClass("com.revenuecat.purchasesunity.PurchasesWrapper"))
+        {
+            purchases.CallStatic("setPhoneNumber", phoneNumber);
+        }
+    }
+
+    public void SetDisplayName(string displayName)
+    {
+        using (var purchases = new AndroidJavaClass("com.revenuecat.purchasesunity.PurchasesWrapper"))
+        {
+            purchases.CallStatic("setDisplayName", displayName);
+        }
+    }
+
+    public void SetPushToken(string token)
+    {
+        using (var purchases = new AndroidJavaClass("com.revenuecat.purchasesunity.PurchasesWrapper"))
+        {
+            purchases.CallStatic("setPushToken", token);
+        }
+    }
+
+    private class DictionaryRequest
+    {
+        public string[] keys;
+        public string[] values;
+
+        public DictionaryRequest(Dictionary<string, string> dictionary) {
+            keys = new string[dictionary.Keys.Count];
+            values = new string[dictionary.Values.Count];
+        
+            dictionary.Keys.CopyTo(keys, 0);
+            dictionary.Values.CopyTo(values, 0);
+        }
+
+    }
```

Comment:
How can we extract this to be used in the WrapperIOS too? I don't want to make it public and expose it.

Reply:
does this create its own assembly? if so, moving it to an `internal` class should do the trick, right?

Reply:
I don't remember what this was. What was it?

Reply:
I believe it was a private class

Reply:
or maybe a private method